hypothesis-python/src/hypothesis/internal/compat.py

```diff
@@ -46,6 +46,31 @@ def get_args(
         return ()
 
 
+try:
+    from typing import get_origin as get_origin
+except ImportError:
+    # remove at Python 3.7 end-of-life
```

Comment:
Note to self: we have a pretty similar helper in `strategies/_internal/types.py`, which dates back to Python 3.4 or so.  We (I) should try porting over to the stdlib version soon and see what breaks.